web-dev/src/App.js

```diff
@@ -13,38 +13,40 @@ import LandingPage from "./components/LandingPage";
 import LoginPage from "./components/LoginPage";
 import RegistrationPage from "./components/RegistrationPage";
 import UserComponent from "./components/RegistrationPage/UserComponent";
-
+import PrivacyScreen from "./components/PrivacyPage/index.js"
 // import Netflicks from "./components/index.js";
 
 
 function App() {
-  return (
-    <BrowserRouter>
-      <div className="container-fluid">
-        <Routes>
-          <Route path="/">
-            <Route index element={(<Navigate to="/landingpage" replace={true}></Navigate>)} />
-            <Route path="homepage"
-                   element={<HomePage />} />
-            <Route path="profile"
-                   element={<Profile />} />
-            <Route index path="search"
-                   element={<SearchScreen />} />
-            <Route path="detail/:movieId"
-                   element={<DetailScreen />} />
-            <Route path="landingpage"
-                   element={<LandingPage />} />
-            <Route path="login"
-                   element={<LoginPage />} />
-            <Route path="registration"
-                   element={<RegistrationPage />} />
-            <Route path="registration/select-avatar"
-                   element={<UserComponent />}/>
-          </Route>
-        </Routes>
-      </div>
-    </BrowserRouter>
-  );
+       return (
+              <BrowserRouter>
```

Comment:
VS Code did this. I AM SORRY. I'll take off my auto formatting next time.

Reply:
Resolved the APP issue from main -> privacy-page in order to get correct routing.